controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java

```diff
@@ -334,8 +334,8 @@ public ActivateResult deployApplication(ApplicationId applicationId, ZoneId zone
 
             ConfigServerClient.PreparedApplication preparedApplication;
             DeploymentId deploymentId = new DeploymentId(applicationId, zone);
-            // TODO: Using deploy() only for user tenants in CD for now
-            if (controller.system().equals(SystemName.cd) && deploymentId.applicationId().tenant().value().startsWith(Tenant.userPrefix)) {
+            // TODO: Using deploy() only in CD and in AWS for now
+            if (controller.system().equals(SystemName.cd) && deploymentId.zoneId().region().value().equals("aws-us-east-1a")) {
```

Comment:
did you mean only cd region in aws, or all cd zones + aws zone?

Reply:
crap, will fix